src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2017,6 +1832,18 @@ private void StartAutoEvoPrediction()
         UpdateAutoEvoPrediction(run, Editor.EditedBaseSpecies, cachedAutoEvoPredictionSpecies);
     }
 
+    private void OnEnergyBalanceOptionsChanged()
+    {
+        CalculateEnergyAndCompoundBalance(editedMicrobeOrganelles, Membrane);
+    }
+
+    private void OnResourceLimitingModeChanged(int index)
+    {
+        balanceMode = (ResourceLimitingMode)index;
```

Comment:
Wouldn't it be better to just have a property in the statistics panel that returns the current mode (similarly to `GetBalanceDisplayType`)? That way things don't break in all the editors if the order of GUI items is changed, which seems like a very good protection to have.